src/praisonai-agents/tests/unit/skills/test_capability_validator.py

```
@@ -0,0 +1,293 @@
+            missing_env_vars=["VAR1"],
+            warnings=["Missing tool2"],
+            errors=[]
+        )
+        
+        data = result.to_dict()
+        assert data["skill_name"] == "test-skill"
+        assert data["state"] == "degraded"
+        assert data["satisfied_tools"] == ["tool1"]
+        assert data["missing_tools"] == ["tool2"]
+        assert data["warnings"] == ["Missing tool2"]
+        assert data["is_fully_satisfied"] == False
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `False`; use `not data["is_fully_satisfied"]:` for false checks

Replace with `not data["is_fully_satisfied"]`

(E712)